### DISCUSSION TITLE
PTM 215 ZE does not pair with Zigbee2MQTT / Homeassistant on Raspberry 4

### DISCUSSION THREAD

**Discussion by apazureck** [Koenkk/zigbee2mqtt#13398] [Devices]
Hello,

I am currently trying to pair an EnOcean PTM215ZE Switch with my Homeassistant using the Zigbee2mqtt Plugin.

Sadly it just does not show up in the device list. I tried it using the setup as described here:

https://www.zigbee2mqtt.io/devices/PTM_215ZE.html

I am not seeing any message in the logs. So I am not sure what to do now. Maybe SO has also had some troubles using this switch and can help me. If I find something out on my own I will post it here.

How I did it:

I started like in the description using Button A0 and A1.

A button push is always first pressing the button (for example A0) down and then pressing the power bar. I do this with any button press.

1. Push button for 7s (so pressing the button, after that triggering the powerswitch, releasing after about 8 - 10 s)
2. Push button short < 2s
3. Pushing button for 7s+
4. Pushing the button shortly and watching my Homeassistant / Zigbee2mqtt browser window, if the message for new device pops up. Waiting for 5s and then pushing the button shortly again, to go through the channels. (My Zigbee2mqtt is on Channel 11).

But no matter how often I press, the device does not show up in the list. What am I doing wrong here? Did I misunderstand anything?

I could also upload a video of what I am doing, if it is not clear in my description.

The current Zigbee2mqtt Version is 1.27.0.

Help would be much appreciated.

Cheers!

**Comment by rdumont** [comment]
@apazureck did you ever get it working? I also cannot get the switch to show up on Zigbee2MQTT.

**Comment by SeeDborg** [comment]
I have the exact same problem.
I managed to pair a PTM215Z, but impossible with the ZE.
Please help :(
